### PR TITLE
Optimize js and minor fix

### PR DESCRIPTION
**Pull request by WalterLuis**
## Description

- change comparator '==' to '===' and '!=' to '!==' for indexof() and typeof()
- some double quotes with single quotes
- minor fix to process install

## Changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [X] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: public_html/layouts/resources/app.js [hunk 1]
```diff
@@ -903,45 +903,47 @@ app = {
...
-				'action': 'SaveAjax',
-				'record': userid,
-				'value': serverWidth,
-				'field': 'rowheight'
-			};
-			AppConnector.request(params).then(function () {
-				$(rowType).val(serverWidth);
-			});
-		}
+		if (typeof rowType !== 'undefined') {
```
Comment (poradz) [inline public_html/layouts/resources/app.js]: rowType is always defined, wrong formatting below

Comment (WalterLuis) [reply public_html/layouts/resources/app.js]: this erros happend in process installation

firefox_2018-04-27_11-52-02

**Comment by paula-w** [comment]
Thank you! Great work, we really appreciate your contributions!

**Comment by WalterLuis** [comment]
thanks to you! 😄

**Pull request merged**